frontend/src/app/shared/components/storages/location-picker-modal/location-picker-modal.component.html

```diff
@@ -72,6 +72,16 @@
   </div>
 
   <div class="spot-action-bar">
+    <div class="spot-action-bar--left">
+      <button
+        type="button"
+        class="button spot-action-bar--action"
+        (click)="newFolderDialog()"
```

Comment:
🟡 I have a naming issue here.

`newFolderDialog` is not what really happens here. It is instead a `createFolderAndEnter()`.